.github/workflows/lint-workflow-files.yml

```diff
@@ -13,4 +13,4 @@ jobs:
         uses: actions/setup-node@v1
 
       - name: Run spectral
-        run: npx @stoplight/spectral lint .github/workflows/*.yml --ignore-unknown-format --verbose
+        run: npx @stoplight/spectral lint [.github/workflows/*.yml] --ignore-unknown-format --verbose
```

Comment:
📣 This prevent the check to happen.

Reply:
I may be wrong, please confirm it still checks and if so, please freeze spectral's version.

Reply:
Actually I'd question the use of this github action. Move it to a git pre commit hook instead...

Reply:
I agree with you. It bugged me more than once already. 
I've removed it from GitHub CI and added it to the pre-commit action.